Makefile

```diff
@@ -0,0 +1,211 @@
+BASE_PACKAGES := utils db settings visualization
+
+MODELS := biocro clm45 dalec ed gday jules linkages \
+	      lpjguess maat maespa preles sipnet
+
+
+MODULES := allometry assim.batch assim.sequential benchmark \
+	       data.atmosphere data.hydrology data.land \
+	       data.mining data.remote emulator meta.analysis \
+	       photosynthesis priors rtm uncertainty
+
+.PHONY = all
+
+all: ${BASE_PACKAGES} ${MODELS} ${MODULES}
+
+#### DEPENDENCY MAPPING ####
+
+${MODELS}: utils data.atmosphere
+
+#### BASE PACKAGES ####
+
+utils: .install.utils
```

Comment:
Because an R package installation has no explicit target. This is a `make` hack whereby, after a successful install of an R package, I make a small text file (`.install.*`) that indicates when the package was installed.

Dependencies are installed automatically by `devtools::install()`, so we shouldn't need to list them explicitly anywhere except each package's `DESCRIPTION`. At least that's the theory -- I'm testing it out now.